doc/lua_api.txt

```diff
@@ -1777,6 +1777,13 @@ supported. It is a string, with a somewhat strange format.
 Spaces and newlines can be inserted between the blocks, as is used in the
 examples.
 
+Position and size units are inventory slots, `X` and `Y` position the formspec
+element relative to the top left of the menu and `W` and `H` are its width and
```

Comment:
`of the menu or container. ``W`` and ``H`` are its width and height values.`

Reply:
https://github.com/minetest/minetest/pull/7315/commits/550bb6348a557986f997e5be782f0128f965a56d
I didn't put ` `` ` instead of `` ` `` around W and H.